### ISSUE TITLE
inspector-image subject / unable to submit audits

### ISSUE THREAD

**Issue by specest** [01-edu/public#1866]
inspector-image
---

Cannot submit audit successfully. On submitting, audit xp from auditee (me) is subtracted and xp for auditor is added, but the system still requires 5 audits, including from the auditor who's code was submitted. Tried submitting the code 6 time. Later discovered that I had audit xp deducted for each submission.

Also cannot find audit markdown

 From broswer network console:
> Request URL: https://01.kood.tech/markdown/root/public/subjects/cybersecurity/inspector-image/audit/README.md
> Request Method: GET
> Status Code: 400

**Comment by freeworldwhoo** [comment]
hello @specest we are looking into this we will get back to you as soon as possible

**Comment by specest** [comment]
Any updates on progress? It's been almost 2 weeks since I first mentioned this issue to kood/Jõhvi and submitted an issue here. I've been unable to progress for 2 weeks and my 10 auditors have been stuck, unable to do any audits. Tomorrow morning I am assigned new auditors, which means I will have to start the auditing process from the beginning. This issue is negatively affecting a lot of people.

**Comment by freeworldwhoo** [comment]
Greetings @specest, our team is actively working towards resolving this issue as quickly as possible. . We understand the urgency of the situation and are committed to ensuring a prompt resolution. Thank you for your patience and understanding.

**Comment by specest** [comment]
I took a look at the requests using burp suite interceptor and found that this is the offending request when submitting the audit:

```
GET /api/validation/johvi/div-01/inspector-image?grade=NaN&code=pwnfv&auditId=31485&eventId=20&groupId=5332&feedback=%7B%7D HTTP/2
Host: 01.kood.tech
Sec-Ch-Ua: "Chromium";v="111", "Not(A:Brand";v="8"
...
```

Maybe this will help you to track down the bug.

When I changed the grade value from NaN to either 1 or 2 (both worked, I didn't try anything else) the audit succeeded. 

This way I managed to pass the audits before they expired, however the problem still remains for the rest of the students. Also I gained audit "down" xp 13 times by trying to submit the audit codes unsucessfully before trying this. Is there any way to resolve this and remove the excessive negative ("down") audit xp?
